src/lightning/pytorch/strategies/xla.py

```diff
@@ -130,13 +130,6 @@ def _setup_model(self, model: Module) -> Module:  # type: ignore
     def distributed_sampler_kwargs(self) -> Dict[str, int]:
         return dict(num_replicas=self.world_size, rank=self.global_rank)
 
-    @property
-    def is_distributed(self) -> bool:
```

Comment:
digged in parents and `DDPStrategy` has
```py
    @property
    def is_distributed(self) -> bool:
        return True
```

Reply:
It is removed too

Reply:
hmmm, then we shall really deprecate it at least in `DDPStrategy`

Reply:
Done